lib/Driver/ToolChains.cpp

```diff
@@ -1182,13 +1212,9 @@ toolchains::GenericUnix::constructInvocation(const LinkJobAction &job,
         Twine("@") + OutputInfo.getPrimaryOutputFilename()));
   }
 
-  // Add the linker script that coalesces protocol conformance sections.
-  Arguments.push_back("-Xlinker");
-  Arguments.push_back("-T");
-    
-  // FIXME: This should also query the abi type (i.e. gnueabihf)
   Arguments.push_back(context.Args.MakeArgString(
```

Comment:
Wow. I'm not convinced this ever worked. The .ld script should also have a -Xlinker.